### PR TITLE
Add org option to flutter create

### LINKED ISSUE #9994: flutter create --plugin should ask for value of com.yourcompany
`flutter create --plugin` uses the placeholder `com.yourcompany` in many, many, many places. It's a huge pain to change it to the correct value everywhere without missing a spot. It would be nice if `flutter create --plugin` would (optionally) ask for the actual value you want to use and use that to populate the template. 

/cc @mravn-google @mit-mit

### PR DESCRIPTION
**Pull request by mravn-google**
Fixes #9994

### CODE REVIEW

File: packages/flutter_tools/lib/src/commands/create.dart [hunk 1]
```diff
@@ -47,6 +47,12 @@ class CreateCommand extends FlutterCommand {
       help: 'The description to use for your new flutter project. This string ends up in the pubspec.yaml file.'
     );
     argParser.addOption(
+      'org',
+      defaultsTo: 'com.yourcompany',
+      help: 'The organization responsible for the new flutter project in reverse domain name notation.\n'
```
Comment (mit-mit) [inline packages/flutter_tools/lib/src/commands/create.dart]: s/the new flutter project/the Flutter project/

**Review by mit-mit — APPROVED** [review]
lgtm, one nit comment

**Review by jakobr-google — APPROVED** [review]
LVGTM

**Review by szakarias — APPROVED** [review]
LGTM :-)

**Comment by Hixie** [comment]
We should probably use com.example rather than com.yourcompany.

**Comment by mravn-google** [comment]
Yeah, I was reluctant to change the behavior of `flutter create` with no `--org` option. If it is used on an existing project with missing files, using a new org may lead to trouble. But perhaps we can ignore that risk.

**Comment by Hixie** [comment]
Ideally we would detect the previously-used name and reuse that if you omit it the second time, or some such. That may be too hard though?

**Comment by mravn-google** [comment]
https://github.com/flutter/flutter/issues/10329

**Pull request merged**